src/utils/dbmlParser/dbmlToJson.js

```diff
@@ -130,13 +130,14 @@ exports.prepareEnumData = applications => {
         let enumData = '';
 
         for (let i = 0; i < applicationCount; i++) {
-            if (applications[i].clientFramework === undefined || applications[i].clientFramework === null) {
+            // loose equality, check for null and undefined.
+            if (applications[i].clientFramework == null && applications[i].dbmlData != null && applications[i].dbmlData !== '') {
```

Comment:
rather than `applications[i].clientFramework == null` check, you can check for service framework to be springboot right.